airlock_processor/shared_code/blob_operations.py

```diff
@@ -35,16 +36,18 @@ def copy_data(source_account_name: str, destination_account_name: str, request_i
             if found_blobs > 0:
                 msg = "Request with id {} contains more than 1 file. flow aborted.".format(request_id)
                 logging.error(msg)
-                raise AirlockInvalidContainerException(msg)
+                raise TooManyFilesInRequestException(msg)
             blob_name = blob.name
             found_blobs += 1
 
         if found_blobs == 0:
-            logging.info('Request with id %s did not contain any files. flow aborted.', request_id)
+            msg = "Request with id {} did not contain any files. flow aborted.".format(request_id)
+            logging.error(msg)
+            raise NoFilesInRequestException(msg)
 
     except Exception:
         logging.error('Request with id %s failed.', request_id)
-        raise ()
```

Comment:
this line raised a type error instead of the actual exception:
"TypeError: exceptions must derive from BaseException"